packages/inferno-compat/src/index.ts

```diff
@@ -136,7 +136,22 @@ function normalizeFormProps<P>(name: string, props: Props<P> | any) {
     const type = props.type;
     let eventName;
 
-    if (!type || type === 'text') {
+    const validLineInputs = [
```

Comment:
I think this should be hoisted and converted to object literal, so it does not need to use `indexOf` check to minimize run time performance cost.